drivers/input/Kconfig.cf1133

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2023 Kickmaker
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig INPUT_CF1133
+	bool "CF1133 capacitive touch panel driver"
+	default y
+	depends on DT_HAS_SITRONIX_CF1133_ENABLED
+	select I2C
+	help
+	  Enable driver for Sitronix capacitive touch panel
+	  controller. This driver should support CF1133.
+
+if INPUT_CF1133
+
+config INPUT_CF1133_PERIOD
+	int "Sample period"
+	depends on !INPUT_CF1133_INTERRUPT
+	default 10
+	help
+	  Sample period in milliseconds when in polling mode.
+
+config INPUT_CF1133_INTERRUPT
+	bool "Interrupt"
+	default y
+	help
+	  Enable interrupt support (requires GPIO).
```

Comment:
can you move this before `INPUT_CF1133_PERIOD`? Then they are in dependency order. I see other drivers like this, sorry about it I'll look into those

Reply:
Ok done